plugins/basic/js/shotgun/socket_io_manager.js

```diff
@@ -270,6 +270,41 @@ sg_socket_io.SocketManager = new function() {
                 );
             };
 
+            /*
+            Compares two objects for equality.
+
+            :param params: A list containing two objects describing what to
+                compare. Each object should contain three properties: uid,
+                value, and is_wrapped. If is_wrapped is true, the comparison
```

Comment:
Wouldn't it be simpler if it was simply two values in the dict, `value` and `is_wrapped`? Then depending on the flag we know if it's a uid or not.

Reply:
Solid idea, in fact. :) I'll also implement `__cmp__` real quick.

Reply:
I take it back, I won't implement `__cmp__` right now. Was thinking about how it might work and it'll require some thought. If the need arises, we'll spend the time then.